schedule/store.go

```diff
@@ -115,6 +127,48 @@ func (store *Store) FindMany(ctx context.Context, ids []string) ([]Schedule, err
 
 	return result, nil
 }
+
+func (store *Store) FindMany(ctx context.Context, ids []string) ([]Schedule, error) {
+	return store.FindManyTx(ctx, nil, ids)
+}
+
+func (store *Store) FindManyByAssignments(ctx context.Context, tx *sql.Tx, userID string, rotationIDs []string) (scheduleIDs []string, err error) {
```

Comment:
Same as rotations, the signature should be `[]Schedule, error` to reduce round-trips and to match the other methods